icon_search.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script searches for icons on your system by name to help find entries for mapping.yaml
+
+echo -n "Please enter an app to search for: "
+read app # Store the prompted app name
+declare -a paths
+readarray -t paths < <(find /usr/share/icons/ -name *$app*.*) # Make an array storing every result for the given app name
```

Comment:
Maybe we should search in all possible locations (especially the icons folder in the home directory)

Reply:
Do you mean `~/.icons`?

Reply:
`~/.local/share/icons` is the offical location for user-installed icons.